packages/amplify-gen1-codegen-auth-adapter/API.md

```diff
@@ -5,17 +5,30 @@
 ```ts
 
 import { AuthDefinition } from '@aws-amplify/amplify-gen2-codegen';
+import { LambdaConfigType } from '@aws-sdk/client-cognito-identity-provider';
 import { PasswordPolicyPath } from '@aws-amplify/amplify-gen2-codegen';
 import { UserPoolType } from '@aws-sdk/client-cognito-identity-provider';
 
 // @public (undocumented)
 export interface AuthSynthesizerOptions {
+    // Warning: (ae-forgotten-export) The symbol "AuthTriggerConnectionMap" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    authTriggerConnections?: AuthTriggerConnectionMap;
```

Comment:
this warning needs solution.

`AuthTriggerConnectionMap` -> `AuthTriggerConnections` / `AuthTriggerConnectionBy<WhateverIsNameOfKey>` ?

Reply:
Changed name to `AuthTriggerConnectionSourceMap`. Resolved export warning